src/main/java/com/capitalone/dashboard/rest/DashboardController.java

```diff
@@ -481,4 +481,11 @@ public ResponseEntity<List<Dashboard>> myDashboardByTitlePage(@RequestParam(valu
                 .headers(paginationHeaderUtility.buildPaginationHeaders(pageDashboardItems))
                 .body(pageDashboardItems.getContent());
     }
+
+    @RequestMapping(value = "/dashboard/removeWidgetDuplicates", method = DELETE)
+    public ResponseEntity<String> removeWidgetDuplicates(@RequestParam(value="title", required = false)String title,
+    @RequestParam(value="dryRun", required = false) boolean dryRun){
```

Comment:
dryRun should be a required field for each run, which helps to be aware on deletion

Reply:
Agree. Either that, or default to TRUE so it only actually runs if explicitly stated